doc/application/index.rst

```diff
@@ -965,13 +965,13 @@ hardware. Follow these instructions to run an application via QEMU:
 
    .. code-block:: console
 
-      west build -t run
+      west build -t run_qemu
 
    or
 
    .. code-block:: console
 
-      ninja run
+      ninja run_qemu
```

Comment:
Why this change.

Let's keep the info that users can still use `ninja run` and have the default emulator used.
And then a note that a specific emulator can be chosen with `ninja run_<emulator>`.